paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/CatalogTableITCase.java

```diff
@@ -53,8 +53,9 @@ public void testSnapshotsTable() throws Exception {
         sql("INSERT INTO T VALUES (3, 4)");
 
         List<Row> result = sql("SELECT snapshot_id, schema_id, commit_kind FROM T$snapshots");
-        assertThat(result)
-                .containsExactlyInAnyOrder(Row.of(1L, 0L, "APPEND"), Row.of(2L, 0L, "APPEND"));
+
+        // check ordered snapshots.
+        assertThat(result).containsSequence(Row.of(1L, 0L, "APPEND"), Row.of(2L, 0L, "APPEND"));
```

Comment:
Should be `containsExactly`?
`containsSequence` verifies that the actual group contains the given sequence in the correct order:
```
   * Iterable&lt;Ring&gt; elvesRings = newArrayList(vilya, nenya, narya);
   *
   * // assertions will pass
   * assertThat(elvesRings).containsSequence(vilya, nenya)
   *                       .containsSequence(nenya, narya);
```

Reply:
Got, `containsExactly` check the correctness and order, thanks.